### PR TITLE
FIX: futureproofed the logout button a bit to redirect on click

### PR DESCRIPTION
**Pull request by Kayahs**

### CODE REVIEW

File: packages/client/src/Play-Game/CreateGamePage.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const CreateGamePage = () => (
       mutation={gql`
         mutation($createMatch: NewMatchInput!) {
           createMatch(input: $createMatch) {
-            match_id
+            id
```
Comment (VPaulger) [inline packages/client/src/Play-Game/CreateGamePage.js]: This is incorrect and will break create game functionality.

File: packages/client/src/Play-Game/CreateGamePage.js [hunk 2]
```diff
@@ -76,6 +76,9 @@ const CreateGamePage = () => (
               values = {
                 ...values,
               }
+              // createMatch({ variables: {
+              //   match: values
+              // } })
```
Comment (VPaulger) [inline packages/client/src/Play-Game/CreateGamePage.js]: These have been deleted previously. Please delete and re-push.

File: packages/client/src/Play-Game/CreateGamePage.js [hunk 3]
```diff
@@ -152,6 +155,7 @@ const CreateGamePage = () => (
                       width: '75%',
                       marginTop: 50,
                     }}
+                    // onClick={alert("Submitting Scores!")}
```
Comment (VPaulger) [inline packages/client/src/Play-Game/CreateGamePage.js]: This has been deleted previously. Please delete and re-push.

File: packages/client/src/Play-Game/CreateGamePage.js (hunk 1 above)
Comment (Kayahs) [reply packages/client/src/Play-Game/CreateGamePage.js]: fixed.

**Pull request merged**